distribution/openrct2.d.ts

```diff
@@ -2495,11 +2495,30 @@ declare global {
          * Highest drop height in height units. Use `context.formatString()` to convert into metres/feet. Ex: `formatString('{HEIGHT}', ride.highestDropHeight)`.
          */
         readonly highestDropHeight: number;
+		
+		 /**
+         * The current breakdown of the ride.
+         */
+        readonly breakdown: BreakdownType;
+		 
+		/**
+         * Set a breakdown on a ride.
+         * @param breakdown The type of breakdown to set.
+         */
+        setBreakdown(BreakdownType: breakdown): void;
```

Comment:
```suggestion
        setBreakdown(breakdown: BreakdownType): void;
```

Almost missed this, this should be the other way around.